### PR TITLE
pull request from rlottie-2 team

### PR DESCRIPTION
**Pull request by innoinho**
made 10 methods into rottieWasm.cpp

### CODE REVIEW

**Review by rlottie — CHANGES_REQUESTED** [review]
cleanup the patches .. squash them to a single commit . add only English description both in commit message as well as comments .

File: 멘토링/0915.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## 팀 소개
```
Comment (hermet) [inline 멘토링/0915.md:1]: Doc in Korean is bad to oversea engineers.
This documentation shouldn't be involoved in this patch.

File: src/wasm/rlottiewasm.cpp [hunk 2]
```diff
@@ -7,16 +7,16 @@
...
...\":[-17]},{\"t\":134.400390625}],\"ix\":10},\"p\":{\"a\":0,\"k\":[-151.699,7.848,0],\"ix\":2},\"a\":{\"a\":0,\"k\":[16.5,-109.52,0],\"ix\":1},\"s\":{\"a\":0,\"k\":[131.332,131.332,100],\"ix\":6}},\"ao\":0,\"shapes\":[{\"ty\":\"gr\",\"it\":[{\"ind\":0,\"ty\":\"sh\",\"ix\":1,\"ks\":{\"a\":0,\"k\":{\"i\":[[-0.618,-7.395],[-2.323,-2.379],[-2.647,2.438]],\"o\":[[6.75,80.75],[1.66,1.7],[26.148,-64.813]],\"v\":[[-1.601,-210.907],[9.521,-107.006],[20.155,-104.319]],\"c\":true},\"ix\":2},\"nm\":\"Path 1\",\"mn\":\"ADBE Vector Shape - Group\",\"hd\":false},{\"ty\":\"fl\",\"c\":{\"a\":0,\"k\":[0.149019607843,0.132484795065,0.111034221275,1],\"ix\":4},\"o\":{\"a\":0,\"k\":100,\"ix\":5},\"r\":1,\"nm\":\"Fill 1\",\"mn\":\"ADBE Vector Graphic - Fill\",\"hd\":false},{\"ty\":\"tr\",\"p\":{\"a\":0,\"k\":[0,0],\"ix\":2},\"a\":{\"a\":0,\"k\":[0,0],\"ix\":1},\"s\":{\"a\":0,\"k\":[100,100],\"ix\":3},\"r\":{\"a\":0,\"k\":0,\"ix\":6},\"o\":{\"a\":0,\"k\":100,\"ix\":7},\"sk\":{\"a\":0,\"k\":0,\"ix\":4},\"sa\":{\"a\":0,\"k\":0,\"ix\":5},\"nm\":\"Transform\"}],\"nm\":\"Shape 1\",\"np\":3,\"cix\":2,\"ix\":1,\"mn\":\"ADBE Vector Group\",\"hd\":false}],\"ip\":0,\"op\":141.6,\"st\":0,\"bm\":0}],\"markers\":[]}";
```
Comment (hermet) [inline src/wasm/rlottiewasm.cpp:10]: any reason you removed this?

**Review by hermet — CHANGES_REQUESTED** [review]
You can extend the functionalities, but do you really need to modify original behaviors?

**Comment by smohantty** [comment]
Not needed anymore

**Pull request closed without merging**